project-addons/custom_emails_company/models/invoice.py

```diff
@@ -7,31 +7,31 @@ class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
 
     def action_invoice_open(self):
+        for invoice in self:
+            res = super(AccountInvoice, invoice).action_invoice_open()
```

Comment:
El super ponlo arriba del for, no ha motivo para que se tenga que ejecutar unitariamente

Reply:
Hecho. Perdona, no sabia si ponerlo dentro o fuera.